errdefs/is.go

```diff
@@ -1,78 +0,0 @@
-package errdefs
-
-import (
-	"context"
-	"errors"
-
-	cerrdefs "github.com/containerd/errdefs"
-)
-
-// IsNotFound returns if the passed in error is an [ErrNotFound],
-//
-// Deprecated: use containerd [cerrdefs.IsNotFound]
-var IsNotFound = cerrdefs.IsNotFound
```

Comment:
We should keep this part (removing the aliases) separate (separate PR), because there will still be external consumers.

Reply:
GitHub is good at hiding things!

On that matter; it _might_ be worth splitting the diff into smaller commits (e.g. per top-level directory, or smaller groups of packages) to help review-ability.